autogpt_platform/backend/backend/data/graph.py

```diff
@@ -392,6 +395,10 @@ def starting_nodes(self) -> list[NodeModel]:
             if node.id not in outbound_nodes or node.id in input_nodes
         ]
 
+    @property
+    def webhook_input_node(self) -> NodeModel | None:  # type: ignore
+        return super().webhook_input_node  # type: ignore
```

Comment:
what's the point of this ?

Reply:
`Graph.webhook_input_node` gives a `Node`, which does not have `webhook_id`. But [here](https://github.com/Significant-Gravitas/AutoGPT/pull/10418/files#diff-5ce1ff827d96e3a774b1fa11dac75c89e0f51a4eca83386f7a7f26b9df2615a1R171), I need `graph.webhook_input_node.webhook_id`. So I need to redefine `webhook_input_node` here with the right return type `NodeModel`.

Reply:
done